scripts/index.js

```diff
@@ -0,0 +1,99 @@
+const state = {
+    currentTemp: parseInt(document.getElementById("temp_val").textContent)
+} 
```

Comment:
This works, but there's an off-chance that the JavaScript file would load before the HTML, in which case this would set the `currentTemp` to `undefined`.